sdk/resourcemanager/azure-resourcemanager-keyvault/src/main/java/com/azure/resourcemanager/keyvault/models/ManagedHsm.java

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.resourcemanager.keyvault.models;
+
+import com.azure.core.annotation.Fluent;
+import com.azure.resourcemanager.keyvault.KeyVaultManager;
+import com.azure.resourcemanager.keyvault.fluent.models.ManagedHsmInner;
+import com.azure.resourcemanager.resources.fluentcore.arm.models.GroupableResource;
+import com.azure.resourcemanager.resources.fluentcore.collection.SupportsListingPrivateLinkResource;
+import com.azure.resourcemanager.resources.fluentcore.collection.SupportsUpdatingPrivateEndpointConnection;
+import com.azure.resourcemanager.resources.fluentcore.model.Refreshable;
+
+import java.time.OffsetDateTime;
+import java.util.List;
+
+/** An immutable client-side representation of an Azure Managed HSM. */
+@Fluent
+public interface ManagedHsm
+    extends GroupableResource<KeyVaultManager, ManagedHsmInner>, Refreshable<ManagedHsm>,
+    SupportsListingPrivateLinkResource,
+    SupportsUpdatingPrivateEndpointConnection {
+
+    /** @return the AAD tenant ID that should be used for authenticating requests to the managed HSM */
+    String tenantId();
+
+    /** @return Managed HSM SKU */
+    ManagedHsmSku sku();
+
+    /** @return initial administrators object ids for this managed hsm pool */
+    List<String> initialAdminObjectIds();
```

Comment:
What is this initial admin object ID?

Reply:
From their doc:
> When a managed HSM is created, the requestor also provides a list of data plane administrators (all [security principals](https://learn.microsoft.com/en-us/azure/role-based-access-control/overview#security-principal) are supported). Only these administrators are able to access the managed HSM data plane to perform key operations and manage data plane role assignments (Managed HSM local RBAC).

I'll add it to the javadoc.